server/pbench/lib/config/pbench-server.cfg.example

```diff
@@ -11,6 +11,13 @@ pbench-local-dir = /srv/pbench-local
 # Backup dir may be remote NFS mounted
 pbench-backup-dir = /mnt/pbench.archive.backup
 
+[aws-s3]
+endpoint_url =
+aws_access_key_id =
+aws_secret_access_key =
```

Comment:
Can we drop the "`aws_`" prefixes here since this is not really about Amazon Web Services?

Reply:
Done